packages/interface/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### Breaking Changes
```

Comment:
I'm not entirely sure it's helpful when publishing with `dist-tag=wp-6.7`. The version bump will be from 6.8.5 to 6.8.6:

https://github.com/WordPress/gutenberg/blob/6235b5d3849b507ee48c44f7c5ab8b0b6dabab61/packages/interface/package.json#L3

The changelog won't get updated. So if we want to keep any log, it should be listed as `Bug Fix` under `6.8.6` section.